libraries/src/Form/Field/SchemaorgComponentSectionsField.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Joomla\CMS\Form\Field;
+
+use Joomla\CMS\Factory;
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Schemaorg\SchemaorgServiceInterface;
+
+// phpcs:disable PSR1.Files.SideEffects
+\defined('JPATH_PLATFORM') or die;
+// phpcs:enable PSR1.Files.SideEffects
+
+/**
+ * Form Field class for the Joomla Framework.
+ *
+ * @since  4.0.0
+ */
+class SchemaorgComponentSectionsField extends ComponentsField
+{
+    /**
+     * The form field type.
+     *
+     * @var     string
+     * @since  4.0.0
```

Comment:
```suggestion
     * @since  __DEPLOY_VERSION__
```

This way it will automatically get the correct version when it is accepted, merged and released. Sorry your mentors didnt explain this